apps/directory/src/stores/biobanksStore.js

```diff
@@ -73,7 +65,7 @@ export const useBiobanksStore = defineStore("biobanksStore", () => {
       .flatMap((biobank) =>
         biobank.collections.filter((collection) => !collection.withdrawn)
       ).length;
-    return totalCount - biobankCardsSubcollectionCount.value;
+    return totalCount;
```

Comment:
This should not only by total count, but you must subtract the sub collection count. Joris fixed this a while ago, because now the total amount of collections is including the sub collection.